engine/src/main/java/org/terasology/physics/bullet/BulletPhysics.java

```diff
@@ -597,7 +620,11 @@ private synchronized void processQueuedBodies() {
         }
         while (!removalQueue.isEmpty()) {
             BulletRigidBody body = removalQueue.poll();
+            if(body.isDisposed) {
```

Comment:
Trivial: code style (missing space) - more of them too, won't comment on each, but may want to run the Checkstyle plugin on each modified file to catch these :-)